src/core/CommandManager.ts

```diff
@@ -28,18 +28,44 @@ export class CommandManager {
                 return "Sensitivity is not in the valid range of 0 to 10";
             }
         }));
+
         this.commands.push(new Command('resetSettings', (): string => {
             SettingsManager.reset();
             SettingsManager.write();
             return "Settings have been reverted to their default states";
         }));
+
+        this.commands.push(new Command('crosshairColor', (args: string[]): string => {
+            if (!(args[1] && args[2] && args[3])) {
+                return 'Not an rgb value'
+            }
+
+            for (let i = 1; i < args.length; i++) {
+                if(Number(args[i]) < 0 || Number(args[i]) > 255) {
+                    return args[i] + ' is not in range 0-255';
+                }
+            }
+
+            SettingsManager.settings.crosshairColor = '#' + componentToHex(Number(args[1])) + componentToHex(Number(args[2])) + componentToHex(Number(args[3]));
+            SettingsManager.settings.crosshairColorFlashing = invertRGB(Number(args[1]), Number(args[2]), Number(args[3]));
```

Comment:
What if you used CSS style color instead of/as well as R, G, B? It would be fairly easy to implement. That way you could type /crosshaircolor red and it'd know what you're talking about. A hex code could also work.  CSS color can be passed directly to the canvas like how you're doing it now. 

Example validation code
```javascript

function isColor(strColor) {
    const s = new Option().style;
    s.color = strColor;
    // If the color is recognized, s.color won't be empty
    return s.color !== '';
}

console.log(isColor('red')); // true
console.log(isColor('#ff0000')); // true
console.log(isColor('invalidColorName')); // false
```